app/components/ak-radio/index.ts

```diff
@@ -17,7 +37,7 @@ export default class AkRadioComponent extends Component {
   }
 
   get checked() {
-    if (this.args.checked) {
+    if (this.args.checked !== undefined) {
```

Comment:
use this `if (!isEmpty(this.args.checked)) {` it will check for empty string and null too

Reply:
Done